pkg/controller/control.go

```diff
@@ -737,6 +737,13 @@ func (c *Controller) Start(volumeSize, volumeCurrentSize int64, addresses ...str
 			continue
 		}
 
+		if len(addresses) == 1 {
+			err = newBackend.ResetRebuild()
```

Comment:
Can we add a comment explaining the purpose of the reset of rebuilding flag for single replica volume?